### PR TITLE
PR: Add some changes necessary for versionner

### PR DESCRIPTION
**Pull request by rlaverde**

### CODE REVIEW

File: {{ cookiecutter.repo_name }}/.gitattributes [hunk 1]
```diff
@@ -3,3 +3,5 @@
 meta.yaml text eol=lf
 build.sh text eol=lf
 bld.bat text eol=crlf
+
+{{ cookiecutter.project_name }}/_version.py export-subst
```
Comment (goanpeca) [inline {{ cookiecutter.repo_name }}/.gitattributes:8]: if versioneer?

File: {{ cookiecutter.repo_name }}/MANIFEST.in [hunk 2]
```diff
@@ -2,3 +2,5 @@ include CHANGELOG.md
 include CONTRIBUTORS.md
 include LICENSE.txt
 include README.rst
+include versioneer.py
```
Comment (goanpeca) [inline {{ cookiecutter.repo_name }}/MANIFEST.in:7]: if versioneer ?

**Review by goanpeca — CHANGES_REQUESTED** [review]

**Comment by goanpeca** [comment]
Rebase?

**Comment by rlaverde** [comment]
> Rebase?

Yes, I also add hooks and version file, this is ready :smile:

File: hooks/post_gen_project.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+PROJECT_DIRECTORY = os.path.realpath(os.path.curdir)
+
+
+def remove_file(file_name):
+    if os.path.exists(file_name):
+        os.remove(file_name)
+
+
+def remove_version_file():
+    """Removes the _version file if versionner pis pgoing to be used"""
```
Comment (goanpeca) [inline hooks/post_gen_project.py]: . at the end

File: hooks/post_gen_project.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    file_name = os.path.join(PROJECT_DIRECTORY,
+                             '{{ cookiecutter.project_name }}/_version.py')
+    remove_file(file_name)
+
+
+def install_versioneer():
+    call(['versioneer', 'install'])
```
Comment (goanpeca) [inline hooks/post_gen_project.py:30]: docstring

File: hooks/post_gen_project.py (hunk 3 above)
Comment (goanpeca) [reply hooks/post_gen_project.py]: Plus fix the writing.

**Review by goanpeca — CHANGES_REQUESTED** [review]

**Review by goanpeca — APPROVED** [review]

**Comment by goanpeca** [comment]
Is this ready @rlaverde?

**Comment by rlaverde** [comment]
> Is this ready @rlaverde?

Yes, merging now

**Pull request merged**